project1/application.py

```diff
@@ -51,13 +53,25 @@ def authenticate():
     Registartion.query.all()
```

Comment:
can you explain the  use of this line?

Reply:
I had written this line to check  the table data is been retrieved. Coming to this command it returns the primary keys of the table

Reply:
when it returns dont you need to store it anywhere?

Reply:
ok